composer.json

```diff
@@ -1,34 +1,20 @@
 {
     "name": "zendframework/zend-developer-tools",
     "description": "Module for developer and debug tools for use with zend-mvc applications.",
-    "type": "library",
     "license": "BSD-3-Clause",
     "keywords": [
+        "zendframework",
         "debug",
         "developer",
-        "zf2",
+        "zf",
         "module"
     ],
-    "homepage": "https://github.com/zendframework/ZendDeveloperTools",
-    "authors": [
-        {
-            "name": "Evan Coury",
-            "email": "me@evancoury.com",
-            "homepage": "http://blog.evan.pro/"
-        },
-        {
-            "name": "Eric Boh",
-            "email": "cossish@gmail.com"
-        }
-    ],
-    "extra": {
-        "branch-alias": {
-            "dev-master": "1.1-dev",
-            "dev-develop": "1.2-dev"
-        },
-        "zf": {
-            "module": "ZendDeveloperTools"
-        }
+    "support": {
+        "issues": "https://github.com/zendframework/zend-developer-tools/issues",
+        "source": "https://github.com/zendframework/zend-developer-tools",
+        "rss": "https://github.com/zendframework/zend-developer-tools/releases.atom",
```

Comment:
`s/zend-developer-tools/ZendDeveloperTools/` and in many other places unless you are going to rename the repo?

Reply:
No, this was just from running the zf-maintainer tool. Fixed and pushed now.

Reply:
Yeh, the tool assumes that repository name is the same as package name defined in `composer.json`.